### DISCUSSION TITLE
PyBoard will not connect after flashing

### DISCUSSION THREAD

**Discussion by garryp4** [micropython/micropython#14272] [STM32 / Pyboard]
I'm learning MicroPython so got a v1.1 PyBoard and started the tutorial found here:
https://docs.micropython.org/en/latest/pyboard/tutorial/usb_mouse.html#sending-mouse-events-by-hand
I got to "Sending mouse events by hand" and the program would crash at:
hid.send((0, 100, 0, 0)) # (button status, x-direction, y-direction, scroll)

When I plugged the PyBoard in it created Com7 and up to this point everything worked. Once the above line was entered, com7 crashed and com8 was created. Not sure why, but com8 just did not talk to the PyBoard. While waiting fro the PyBoard to show up I looke daround the MicroPython site a bit and found there was current firmware. So I though maybe I just need the newest version. So I followed the instructions here:
https://learn.sparkfun.com/tutorials/how-to-load-micropython-on-a-microcontroller-board/all#pyboard
I loaded WinUSB(v6.1.7600.16385) then flashed PVBV11-2024022-V1.22.2.dfu to the PyBoard. Now when I plug it the PyBoard, the green LED flashes and just sets there. I press the USR button and Com8 is created and on Hterm I get:
MicroPython v1.22.2 on 2024-02-22; PYBv1.1 with STM32F405RG
Type "help()" for more information.
>>>
I can turn on/off the LED's so there is something still working. However, before I flashed it, a flash drive was created when I plug the PyBoard created. I tried to install the PYBCDC.INF driver to Com8 in Device Manager, but Windows does not see the file when using "Update driver". I can see the file in File Manager. 

What did I break?

Thanks

**Comment by IHOXOHI** [comment]
Hey,
I think it's a classic problem of rights... A permanent war...
Have you tried with a SD card?

**Comment by garryp4** [comment]
I do not understand the answer. What am I supposed to do with the SD card?

**Comment by IHOXOHI** [comment]
You have a pybV1.1... So, you can see a place for enter a sd card, you push a sd card inside.
Maybe, you will see the sd like a folder... You create a file named "main.py" with:
from pyb import LED
green = LED(2)
green.on()

You save this file on the sd, and reset (after an umount) the pyb.
If you see the green led...
In this case, you may have more easy if you tyry to use a software like pyboard-rshell on debian, ..., depend of your computer... screen seems to works on the most of them... But, you have an acces to the files on the memory of the pyboard or the sd card only by python commands... Enjoy!

**Reply by garryp4** [reply]
"However, before I flashed it, a flash drive was created when I plug the PyBoard"

There is no longer a drive crated to write to.
